docs/source/guides/network_node/ursula_configuration_guide.rst

```diff
@@ -43,7 +73,7 @@ for more information.
 .. _fund-worker-account:
 
 2. Fund Worker Account with ETH
--------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Ensure that the worker's ethereum account has ETH for transaction gas.
```

Comment:
```suggestion
Ensure that the worker's ethereum account has sufficient ETH to pay the gas for regular transactions, or it may forgo subsidies (inflationary rewards). **Note that reducing the gas costs burdened upon stakers/workers is an active and high-priority area of network development.**
```